### ISSUE TITLE
[director-free] discipline and CEO recusal

### ISSUE THREAD

**Issue by frivoal** [w3c/w3process#275]
As mentioned in an [inline issue](https://w3c.github.io/w3process/director-free/#issue-029fcab7):
> What about questions of disciplinary actions linked to the CEO themself, or to Elected Positions (AB, TAG)? Maybe the CEO should recuse themself for these, and they should be adjudicated by the BoD once we have one.

**Comment by nigelmegitt** [comment]
Seems like a good idea - though the person/people to whom the adjudication will apply should probably not get a choice about recusing themselves.

**Comment by michaelchampion** [comment]
Presumably we can't implement the director-less process until there is a Board of Directors that can hire/fire/discipline the CEO.  So the topic of complaints about the CEO or their decisions probably has to be covered in the Bylaws, not the Process.

**Comment by frivoal** [comment]
The BoD has authority over the CEO, so disciplinary action against the CEO rightly belongs to it once it exists, and with the Steering Committee until then, without the process having to say anything about it.

I'm OK with the process being silent about that.

But the question was not just discipline of the CEO, but also of participants in elected bodies, and whether that too should extraordinary enough that the CEO should recuse themself for these too. If we think that should be handled by the BoD once it exists, we could put in the Steering Committee until then, and update that when the BoD starts being a thing.

Or we can just close this issue with no change, and trust that the CEO will use sufficient caution in cases of discipline involving elected individuals, possibly referring to the Steering Committee / BoD when necessary.

**Comment by michaelchampion** [comment]
> we can just close this issue with no change, and trust that the CEO will use sufficient caution in cases of discipline involving elected individuals, possibly referring to the Steering Committee / BoD when necessary.

That's my preference

**Comment by chaals** [comment]
I think the case of elected representatives is sufficiently rare, and the importance to the members that they can be represented is sufficiently obvious, that a BoD, CEO, Director or Steering committee are not going to do anything that goes far beyond what the members think is reasonable.

There is still a risk that it occurs, but I expect the reaction to be swift enough to deal with it. So I am happy to close the issue without change.

**Comment by avneeshsingh** [comment]
The management of a consortium is accountable to members, and CEO making such decisions for elected representatives of members looks out of the order. Firstly I think that it would be good to implement directorless process along with board of directors. In case we want to move forward with directorless process without BOD then it would be better to have a co-chairs for elected committees (like AB). One chair should come from elected representatives and other chair may be CEO, so that if such action has to be taken for elected members, it could be delegated to the co-chair who comes from elected representatives. Or we can say that the decision should be undivided decision of both the chairs.

**Comment by michaelchampion** [comment]
The only way I can see to organize things going forward is:
Membership ("stockholders") select the Board
The Board hires/fires/disciplines the CEO and perhaps other key executives 
The CEO hires/fires/disciplines the staff

Concerns about staff behavior go to the CEO
Concerns about the CEO behavior (e.g refusal to recuse themselves) go to the Board
Concerns about Board members' behavior go to the rest of the Board
Concerns about the Board's performance on any of its duties go the the Membership

We should try to get as much of this up and running while we still have a Director to appeal if bad things happen that the Board isn't ready to handle.... and to handle any challenges with an interim Board partly selected by the current hosts.

**Comment by dwsinger** [comment]
I agree with Michael; if the CEO is behaving inappropriately, I think appeal to the Board would be in order.

**Comment by avneeshsingh** [comment]
Conceptually this structure is aligned to my thoughts. There are some more details that we need to resolve because it is a consortium. Happy to discuss further in Amsterdam, leaving for airport soon.

**Comment by frivoal** [comment]
Agreed to close in the 2022/10/28 Process-CG meeting.
